services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -580,6 +602,15 @@ export class NoticeOfWorkApplication extends Component {
     return isDisabled;
   };
 
+  setPermitGenerationStep = (step) => {
+    this.setState({ prevStep: step });
+    this.setState({ currentStep: 100, isPermitGeneration: true });
+  };
+
+  returnToPrevStep = () => {
+    this.setState({ currentStep: this.state.prevStep, isPermitGeneration: false });
+  };
+
```

Comment:
this logic is all likely to be removed once the new flow is implemented, correct?

Reply:
I don't know that it will be removed changed probably. I don't know how the draft flow will work with this.